pkg/cmd/root_test.go

```diff
@@ -30,7 +30,7 @@ func TestCommand_no_global_flags(t *testing.T) {
 	p := &test.Params{}
 	pipelinerun := Root(p)
 	out, err := test.ExecuteCommand(pipelinerun)
-	if err == nil {
+	if err != nil {
```

Comment:
is this required?

Reply:
The command at line https://github.com/tektoncd/cli/pull/1010/files#diff-4fd9d4fc993ab96f0674c2725fccc4d1R89 is not returning an error and we need this function.

Reply:
the function `cmd.Help()` at line https://github.com/tektoncd/cli/pull/1010/files#diff-4fd9d4fc993ab96f0674c2725fccc4d1R77 is not returning an error but instead returning `0` status code

Reply:
Thanks for updating this. Could you please change `t.Errorf("test should have command with an error: `subcommand is required` Output: `%s`", out)` to something to convey that no error is expected?